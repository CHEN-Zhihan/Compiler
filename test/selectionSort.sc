array x[10];

for (i = 0; i != 10; ++i;) {
    x[i] = 1000-i;
}

for (i = 0; i != 10; ++i;) {
    min = i;
    for (j = i; j != 10; ++j;) {
        if (x[j] < x[min]) {
            min = j;
        }
    }
    temp = x[min];
    x[min] = x[i];
    x[i] = temp;
}

for (i = 0; i != 10; ++i;) {
    puti(x[i]);
}