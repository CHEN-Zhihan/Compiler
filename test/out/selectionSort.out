991
992
993
994
995
996
997
998
999
1000
